tests/cpp-tests/Resources/ccs-res

```diff
@@ -1 +1 @@
-Subproject commit 892e3cbdc27d24c39c62d700a81be27e593823b7
+Subproject commit 5d65db4c5f18c0df1305ff32b076425ab228cc4a
```

Comment:
I added some ETC format resources to the ccs-res repo. So this submodule change is intended.